Sources/SwifterSwift/SceneKit/SCNVector3Extensions.swift

```diff
@@ -29,6 +29,10 @@ public extension SCNVector3 {
     var length: SceneKitFloat {
         return sqrt(pow(x, 2) + pow(y, 2) + pow(z, 2))
     }
+    
+    func normalize() -> SCNVector3 {
```

Comment:
I feel like it would be "swifter" to use a `var` and call it `normalized`, as this is a non-mutating function.
```suggestion
    var normalized: SCNVector3 {
```